terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars

```diff
@@ -0,0 +1,34 @@
+aws_bucket = "development-eu-west-2.terraform-state.ch.gov.uk"
+remote_state_bucket = "ch-development-terraform-state-london"
+environment = "cidev"
+deploy_to = "development"
+state_prefix = "env:/development"
+aws_profile = "development-eu-west-2"
+kms_alias = "alias/development-eu-west-2/environment-services-kms"
+
+# service configs
+api_local_url = "https://api.cidev.aws.chdev.org"
+authentication_middleware = "on"
+cache_server = "cidev-chs-elasticache.9vgugg.ng.0001.euw2.cache.amazonaws.com:6379"
+cdn_host = "d3uvya5a8a1ncx.cloudfront.net"
+chs_api_key = "OpXFFV77ghkgZlK9U7HcVvaK0rUqKUK0M0AlDi5c"
```

Comment:
Not sure whether this is the correct value or not.

Reply:
i have a feeling that might need to be moved to vault/secret

Reply:
Cannot see it in the profiles or in the vault section of overseas_entities_web, only in locals.

Reply:
Removed this field.